### ISSUE TITLE
homebrew installed printing date and revision

### ISSUE THREAD

**Issue by charlierudolph** [Originate/git-town#631]
@allewun 

Code that checks for manual install
```sh
function is_git_town_installed_manually {
  local base_dir=$(dirname "${BASH_SOURCE[0]}")

  if [ "$(git -C "$base_dir" remote -v | grep -c "https://github.com/Homebrew/homebrew.git (fetch)")" == 0 ]; then
    echo true
  else
    echo false
  fi
}
```
My output from printing the remotes (doesn't match as the `.git` is missing) 
```
> git -C /usr/local/Cellar/git-town/0.7.3/libexec/ remote -v
origin	https://github.com/Homebrew/homebrew (fetch)
origin	https://github.com/Homebrew/homebrew (push)
```

I think we should either
* make the grep less specific as just looking for homebrew should be enough 
* always print the date and revision which is nice to have anyway.

**Comment by allewun** [comment]
For casual users, the revision and date is perhaps too much info? I don't really feel strongly either way. I wonder if homebrew made a recent change, mine is like that too.

**Comment by charlierudolph** [comment]
Its at least nice to have the date so you know how recent the release is from without having to look it up.

**Comment by charlierudolph** [comment]
Actually that date is probably wrong anyway as its from the Homebrew repo. 
Yeah this just needs to be fixed.

**Comment by charlierudolph** [comment]
Fixed in 7672e4b3361db6e72600ec9ead8b2337ab73c4cf
